### DISCUSSION TITLE
Is it possible to use a windows computer as an openthread border router instead of a raspberry pi?

### DISCUSSION THREAD

**Discussion by GRA5S** [openthread/openthread#9747] [Q&A] [answered]
Is it possible?

**Comment by abtink** [comment, marked as answer]
Technically running OpenThread stack on any platform/OS is possible. The OT stack itself is portable.
The requirement is for the platform to provide/implement the defined OpenThread platform APIs ([`include/openthread/platform.h`](https://github.com/openthread/openthread/tree/main/include/openthread/platform). The `ot-br-posix`  provides a posix implemenation of platform layer.
